scopes/component/component/ui/menu/menu.tsx

```diff
@@ -32,13 +36,20 @@ export function Menu({ navigationSlot, widgetSlot, className, host, menuItemSlot
   const { component } = useComponent(host);
   const mainMenuItems = useMemo(() => groupBy(flatten(menuItemSlot.values()), 'category'), [menuItemSlot]);
 
-  if (!component) return <FullLoader />;
+  const versionList =
+    useMemo(
+      () =>
+        component?.tags
+          ?.toArray()
+          .map((tag) => tag?.version?.version)
```

Comment:
filter undefineds?

Reply:
I moved the if further down to be able to use `useMemo` so component might be undefined here

Reply:
but I can also remove the use memo from the version list or extract to function so I will still have the `if(!component)` before the versions

Reply:
fixed